### ISSUE TITLE
disagree with taxon restriction GO:0007610 behavior

### ISSUE THREAD

**Issue by pfey03** [geneontology/go-ontology#12357]
There is a new taxon restriction for GO:0007610 behavior (metazoa). Dictyostelium is a social amoeba, and during development cooperation is essential, but e.g. there are cheaters who preferentially become 'spore' instead of altruistically 'stalk'. Several studies have been published, and I annotated genes involved in cheating (isn't that a behavior?) with GO:0035176 social behavior. 
PMID:18272966
PMID:23910661

Petra

**Comment by dosumis** [comment]
We agreed some time ago to restrict behaviour to things with nervous systems. I stripped out all the plant terms from this branch last year (possibly earlier).  Some had crept back , so I added the constraint to make sure they didn't.  Def of behavior should be updated to reflect this (my bad). 

Seems like in dicty, as in plants, the distinction between development and 'behavior' (in the broad sense) is blurred.  This is made more complicated in dicty by the difficulty in defining organism (single cell, vs slug/fruiting body).

I can see two possible solutions for your use case:

* broaden behavior again - adding a high level division to separate out nervous-system based behaviors from others.
* Find a new home for this type of social interaction between unicellular organisms under multi-organism process.

**Comment by pfey03** [comment]
the Dicty cooperation is not a muti-organism process. It is within the same species typically. Kin recognition also plays into this complex behavior.  Also, this applies solely to the multicellular stages of Dicty. I wouldn't mind calling it something differently for Dicty, e.g. 'social cooperation' comes to mind. Or is this a behavior? 

* Maybe, since GO:0007610 behavior is directly under biological process, maybe that top level, which shouldn't be annotated manually anyway, could be a parent for all and after that we distinguish between nervous system based and others?  

* Or, a new term 'social cooperation' could be part_of sorocarp development? 

But Myxococcus also have this social behavior I think, so it wouldn't be just dicty, theoretically. The first, general solution would be better.

**Comment by dosumis** [comment]
> the Dicty cooperation is not a muti-organism process

multi-organism != multi-species.  The multiple organisms can be of the same species. Pregnancy & mating are both multi-organism processes.

**Comment by pfey03** [comment]
dicty aggregation (GO:0031152 aggregation involved in sorocarp development) is not a child of  GO:0044764 multi-organism cellular process (and thus GO:0098630 aggregation of unicellular organisms), but child of GO:0044707 single-multicellular organism process, via GO:0030587 sorocarp development.

If we decide to put a non restricted behavior branch under multi-organism process, this seems inconsistent. What really is an example of multi-organism cellular process ??

**Comment by dosumis** [comment]
This seems to come down to the fundamental issue of why slime molds are hard to model: What counts an individual organism? Are the single cells, prior to aggregation, individual organisms?  Do they become one organism once they've aggregated?

We mostly call the co-operative 'behavior' (broad sense) of individual cells of a single organism 'development' - which is how sorocarp aggregation is treated. When multiple organism are invovled,we classify as a multi-organism process

> What really is an example of multi-organism cellular process

Biofilm formation
aggregation of unicellular organisms (e.g. floccing)
...

**Comment by dosumis** [comment]
> This seems to come down to the fundamental issue of why slime molds are hard to model: What counts an individual organism? 

This is, of course, also why they are so damn cool.

**Comment by pfey03** [comment]
first of all, 'slime mold' is misleading and an old overcome class name. That's why nowadays they are referred to as 'social amoeba' as this much rather describes what they are. I would think the single cells before aggregation are single cell organism - they can come from one single cell  that eats and divides. Genes have been identified that determine kin recognition. They do not like to mingle with others. When some 'bad' guys of their own mutate and want to cheat, some of those genes mutated also have been identified.

Ok, so what's the solution to our problem? Yes they are cool and have so many disease genes, even form an early form of bipolar epithelium during development - why always throwing Dicty out of the wonderful term families, terms I was so glad to apply in the first place?

**Comment by dosumis** [comment]
I think it's fair to say that In GO, we try to group things that have similar underlying mechanisms. 

Mechanistically, social behaviour controlled by a nervous system is going to be mechanistically very different from the mechanisms involved in co-operation between individual cells during development. So - I think it is justified to keep these separate.

> I  would think the single cells before aggregation are single cell organism.  

And after aggregation, is each cell still its own organism?  This would have to be that case for an annotation to 'social behavior' to be correct: 

* **intraspecies interaction between organisms**: "Any process in which an organism has an effect on an organism of the same species."
  *  < -is_a **social behavior** "Behavior directed towards society, or taking place between members of the same species. Occurs predominantly, or only, in individuals that are part of a group."

I'm happy either way, we just need to be consistent in order to fit this into the GO.

If the individual cells within a fruiting body are considered to be socially co-operating separate organisms, then some other, non-behavior grouping under 'intraspecies interaction between organisms' would work.   This assumption clashes with  the classification of 'sorocarp development' under 'single-organism developmental process', but we can fix that (at least I hope we can, the ontology is very tangled).

How about this:

* <- is_a multi-organism developmental process
    * <- is_a multi-species developmental process
       * <- is_a development involved in symbiotic interaction
    * <- is_a intraspecies multi-organism developmental process
        * <- is_a plantlet formation on parent plant
        * <- is_a sorocarp formation
           *  <- part_of social co-operation between unicellular organisms involved in sorocarp development


* intraspecies interaction between organisms
   * <- is_a social co-operation between organisms
      * <- is_a social co-operation between unicellular organisms
      * <- is_a social co-operation between multicellular organisms
      * <- is_a social behavior

This structure would gives us a way to support annotation to genes involved in social co-operation between bacteria or unicellular fungi during colony formation and between non-metazoan multicellular organisms.

**Comment by jimhu-tamu** [comment]
Independent of the multi vs single organism issue: I would also argue that behavior vs development is a false dichotomy. Chemotaxis has been described as a behaviour for decades. 

http://www.nature.com/nature/journal/v254/n5499/abs/254389a0.html

**Comment by jimhu-tamu** [comment]
Upon review, I'm confused. The term GO:0007610 is just behavior, and it's in go subsets for plants and prokaryotes. I had seen (metazoa) in the original post, but that's not part of the term name or definition.

**Comment by dosumis** [comment]
* If this change sticks: 
  * Agree that definition needs updating .  
  * Subsets are non-semantic, but would clearly need to be adjusted.

> Chemotaxis has been described as a behaviour for decades.
 * GO:chemotaxis is not under behavior - and shouldn't be as it covers chemotaxis of single cells in metazoa, which is certainly *not* behavior.
 * Need to be careful not to confuse terminology issues (which are hard in an ontology covering all of biology) with class definition issues.

I think we can agree that 'behavior mediated by nervous systems'  and the 'response of unicellular organisms to their environment' & each other - are mechanistically *very* different.  We wouldn't expect much share molecular machinery, so grouping/enrichment to behavior nodes that pulls in both uses will be uninformative. In eukaryotes at least,  'response of unicellular organisms to their environment and to each other'  is likely to have more in common with similar processes occurring within metazoans than with nervous system based behavior.  I think we should group accordingly (see chemotaxis).

If we do keep behavior in the general sense - then I'd at least like to have a branch high up into nervous-system based and  non. This will still leave a nasty tangle between development and behavior - certainly for plants.

To decide which option to take, it would help to get a sense of how many and which annotations are be failing due to this restriction for annotations from plants and unicellular organisms.  Jim: do you have any?

**Comment by pfey03** [comment]
> I would think the single cells before aggregation are single cell organism. 

YES

> And after aggregation, is each cell still its own organism? This would have to be that case for an annotation to 'social behavior' to be correct: 

Cells are identical when they eat and divide. Once they starve, they start aggregating  through G-coupled receptor signaling. They do not fuse or divide once development starts, so the cell number stays the same and yes, each cell stays an individual. Cells differentiate/specialize when they start fruiting. However, from the medium slug stage, cells can still be de-differentiated. Also, the cheating shows there is a competition who becomes spore until very late in development.

> intraspecies interaction between organisms: "Any process in which an organism has an effect on an organism of the same species."
< -is_a social behavior "Behavior directed towards society, or taking place between members of the same species. Occurs predominantly, or only, in individuals that are part of a group.

I would say YES

> If the individual cells within a fruiting body are considered to be socially co-operating separate organisms,...

What exactly defines an 'organism'? Is each single cell an organism, even if these cells are identical?

**Comment by jimhu-tamu** [comment]
I don't think we have any currently failing, probably because as you noted, chemotaxis is not a child of behavior. But as you know, the prokaryotic annotations are spotty. There are social bacteria that have issues similar to what Petra is describing for Dicty. 

From wikipedia

> Although there is some disagreement as to how to precisely define behavior in a biological context, one common interpretation based on a meta-analysis of scientific literature states that "behavior is the internally coordinated responses (actions or inactions) of whole living organisms (individuals or groups) to internal and/or external stimuli"[4]

wrt development vs behavior, isn't a lot of circadian clock stuff behavior without developmental differences?

**Comment by pfey03** [comment]
any solution to this in sight? Since it's such a big issue, does the new discussion about 'population of unicellular-organism process terms' help clarifying this?
https://github.com/geneontology/go-ontology/issues/12614

**Comment by pfey03** [comment]
I'd like to close this. 
Here is a proposal for a new term, 'social cooperation'.
I wonder if this could be placed under cell communication
GO:0007154: cell communication 
is_a  GO: NEW social cooperation
part_of non reproductive fruiting body development (parent of 'sorocarp development')

This term should also work for social bacteria, if under cell communication?

Really would be nice to close this!!

**Comment by jimhu-tamu** [comment]
I asked Debby Siegele to look at this

**Comment by dsiegele** [comment]
Summary of this branch of the ontology:
![cell-cell_signaling](https://cloud.githubusercontent.com/assets/12094502/20156734/324bfdae-a697-11e6-8172-d09b84b40739.jpg)

1) GO:0007154 and GO:00007267 are a child and grandchild of 'GO:0044763:single organism process.'  I'm used to thinking about these social processes as occurring within a population of individual organisms, so I would have started off at 'GO:0051704: multi-organism process.'  I think I understand from the earlier discussion that the reason to call it a single organism process is because the populations we're trying to describe are usually clonal? Is this correct? It would be helpful to add usage guidelines to the comments or perhaps modify the definition of single organism process to include clonal populations. 

2) The situation seems to get more complicated when I consider quorum sensing. Quorum sensing is a way of coordinating the behavior of organisms in a population in response to the population density.  Quorum sensing often occurs in a clonal population, which based on the reasoning above would make quorum sensing a 'single organism process.' But currently, the ontology places 'GO:0009372 quorum sensing' as a grandchild of 'multi-organism cellular process.' 'Quorum sensing' has children  'intraspecies quorum sensing' and 'interspecies quorum sensing.' 'Intraspecies quorum sensing' could be considered a 'single organism process,' while 'interspecies quorum sensing' would not. Yet it sort of makes sense to have them near one another in the ontology. 

I will look into 'interspecies quorum sensing' and try to figure out what process it refers to. I know of one example of interspecies communication where bacteria in species A 'eavesdrop' on the quorum signaling of bacteria in species B, and adjust their behavior. But species A isn't using the signal to measure its own cell density, so perhaps this interspecies signaling shouldn't be considered quorum sensing per se. I will find out more about this.

3) I would make the proposed new term 'GO: NEW social cooperation' be a child of 'GO:00007267: cell-cell signaling' rather than a child of 'GO:0007154:cell communication.' Can you explain why you want to make the new term a sibling rather than a child of cell-cell signaling?

Debby

**Comment by pfey03** [comment]
Hi Debby,

Thank for your input and sorry I was quiet for a while, got quite a cold after I came back from LA and just now felling much better. I really appreciate your deep thoughts about this.

1) To  be honest, I was always a bit confused about 'single organism process' and 'multi-organism process'. In Dicty single cells aggregate, but they do not need to be clonal, they just need to be the same genetic make-up for genes that have been identified being surface adaptors for aggregation. Also, in the fruiting body, cells stay separate and differentiate. At the end of differentiation, they cannot be reverted to amoeba, therefore, in the fruiting body, I tend to think of that as a single organism, but I guess technically it could be a multi-organism process.

2) I'll attach the tree for sorocarp development, which is the term I need the 'social cooperation' as a (distant) parent of. In the tree both 'single-organism process' and 'multicellular organismal process' are parents for sorocarp development. Maybe the 'single organism process' should not be there and 'cell communication' should not be the the exclusive grant-child? Why is 'cell communication ' not a child 'single multicellular organism process'? I do not see  'GO:0051704: multi-organism process.' in that tree, and maybe that needs changing?
![sorocarp development](https://cloud.githubusercontent.com/assets/1349602/20365331/2ebf8654-ac0c-11e6-91b8-8f82447c7d5a.png)

In sum, do we need some changes of the organismal upper term structure before we can add 'social cooperation'?

Many thanks again for your input,
Petra

**Comment by pfey03** [comment]
@paolaroncaglia ,

Would you have a look at this as it's now a new term request when you get a chance? From when I added that label during the GO meeting, adding a new term to come to a conclusion of this taxon quagmire. 

Let me know if I should move the latest to a new NTR issue.

Many thanks!!!
Petra

**Comment by dosumis** [comment]
Hi Petra,

Sorry for such long delay. I'll work on this again tomorrow.

**Comment by dsiegele** [comment]
Hi Petra,

I finished writing and grading exam 3 and finally have a few neurons available to think about this topic again.  I'll work on this and post my comments later this afternoon.

Debby

**Comment by dosumis** [comment]
I spent an hour or two looking into the issues on this ticket today.  Rather made my head spin.  I have lots of notes, but need to get them and my thoughts in better order before sharing.

**Comment by dosumis** [comment]
Hi @ukemi 

Could do with some help here. The [class graph](https://github.com/geneontology/go-ontology/issues/12357#issuecomment-261072234) needs some refactoring.  Standout problem: 'sorocarp development' is_a cell communication !!  Wondering if you remember the history here.

Cheers,
David

**Comment by pfey03** [comment]
Hi David, 

> 'sorocarp development' is_a cell communication !! Wondering if you remember the history here.

sorocarp development is_a cellular response to stimulus which is_a cell communication. Seems nothing wrong with this.
petra

**Comment by dosumis** [comment]
*sorocarp development:* The process whose specific outcome is the progression of the sorocarp over time, from its formation to the mature structure. The process begins with the aggregation of individual cells and ends with the mature sorocarp...

*cell communication:* Any process that mediates interactions between a cell and its surroundings. Encompasses interactions such as signaling or attachment between one cell and another cell, between a cell and an extracellular matrix, or between a cell and any other aspect of its environment.

(A process that involves around 100,000 cells aggregating into a slug, migrating and then undergoing morphogenesis and differentiation to form a sorocarp) is_a (process that mediates interactions between *a* cell and its surroundings) ?   

It takes two steps to get to here:

1.  'cellular response to stimulus' is_a 'cell cell communication'  seems to confuse the process of communication with the results.  Open question: how far down a causal chain from the initial stimulus can this go?
2.  'sorocarp development' is_a 'cellular response to starvation'.   While response to starvation is clearly the initiating step, I can't see how we can call it cellular.   Should we consider muscle development a part of the cellular response to mesoderm induction?

**Comment by pfey03** [comment]
>... interactions such as signaling or attachment between one cell and another cell, ..

TRUE

>  (process that mediates interactions between a cell and its surroundings) ?

Not its surroundings, it is signaling between the cells. They don't just signal when they aggregate and stop, there is still signaling while they fruit.

> .. While response to starvation is clearly the initiating step, I can't see how we can call it cellular.

Why not? It is the cells that respond to starvation and then signal and there is further signaling and response to culminate into a fruiting body, still as a downstream effect of the starvation. Before there is a stalk (in slug stage), cells can still be disseminated and fed and start eating and dividing again. Once progression goes too far, the dead stalk cells cannot be reverted. But the whole fruiting business is really just because starvation. No other purpose (that's why I recently asked Tanya to take it out from being under reproduction).

**Comment by dosumis** [comment]
> "It is the cells that respond to starvation and then signal and there is further signaling and response to culminate into a fruiting body, still as a downstream effect of the starvation." 

We could say this about any part of development.  An inductive signal (e.g. mesoderm induction) is a cell-cell signaling event that initiates a long chain of events including morphogenesis, cell migration, more cell signaling, cell adhesion, progressive specification, differentiation and cell death.  By this logic we could classify development of all mesoderm derivatives as cell-cell signaling.   But then all those other events would be parts of cell-cell signaling.

This is instructive:

![image](https://cloud.githubusercontent.com/assets/112839/20557821/0b8f0822-b165-11e6-939d-c689cc88a179.png)

Note that almost everything returned by this query is a part of 'sorocarp development'.  We don't see lots of other developmental processes, despite these being downstream of inductive signals.

I think we need to step back and consider what you want to achieve re the ability to distinguish and group annotations in this branch. It is useful to be able to distinguish annotations to genes involved in cell-cell signaling (secretion of cAMP; cAMP reception and signal transduction) from genes involved in cell migration slug crawling or stalk/spore differentiation, and to group the genes annotated to cell-cell signaling in sorocarp dev with genes annotated to similar signaling processes in other species.  It is also useful to be able to group many or all of these events under a general developmental process term - sorocarp development.

Question: Should the term 'sorocarp development' encompass the cell-cell signaling event(s) that lead to aggregation.   Do you have terms for this already?

(Apologies for yet another side-track on this ticket - but I think it's unavoidable)

**Comment by ukemi** [comment]
I am certainly no expert here.   Based on the biology and the discussion, it seems like perhaps the response is regulating or upstream of the development rather than the development being the response. Looking at the children of the development term, it looks like the development begins with the formation of the aggregate. We have defined that as beginning with the cellular response to starvation. So GO:0031152 starts with the molecular mechanism that detects and responds to starvation, it then continues with the migration of the cells toward one another and the eventual cell cell adhesion that forms the slug. The aggregation process is followed by the development of the slug and finally by culmination. So from a GO development point of view, the identity of the sorocarp is established at the step of responding to starvation. Establishment of identity is not always easy to determine, but that is a tangent. We still discuss that in the office with respect to mouse anatomical structures. Considering all of this, it seems like the development itself is not a cell communication, but part of the development is. The development of the sorocarp itself is not a cellular response to starvation, but from what Petra says above (correct me if I am mistaken), it seems like the DIcty people would consider it a response to starvation. So the cellular response is part of the response of the whole structure. 

Dicty is one of those interesting cases where the lines of the biology get blurred since it switches from ameba to essentially a multicellular organism. But that is one of the cool things about biology isn't it? So when does it make that switch? From the above, it seems like it might be reasonable to consider the switch at the 'cellular response to starvation' step. When the cells do that, it is the first step of them being a multicellular organism rather than single individual cells.

Dicty experts????

**Comment by dosumis** [comment]
> Considering all of this, it seems like the development itself is not a cell communication, but part of the development is

Agreed.

> it seems like it might be reasonable to consider the switch at the 'cellular response to starvation' step. When the cells do that, it is the first step of them being a multicellular organism rather than single individual cells.

I've been thinking that too.  At least, once an aggregate is formed we could consider the aggregate to be one organism, and the development terms apply to the whole structure.  Prior to that we can model as a multi-(unicellular) organism process.  

That still leaves the problem of how to annotate genes whose mutation leads to 'cheating' during sorocarp development.  It seems hard to reconcile this with treating sorocarp development as a single organism process.   We've been discussing 'social co-operation' as an alternative to 'social behavior', but this is still hard to define as a single organism process.  

**social co-operation:** "Communication between organisms resulting participation of all communicating organisms in a multi-organism process that ... "

and then we need a clause the says something about co-operation otherwise the definition equally covers antagonistic multi-organism processes.  One possibility:

"... has an equal chance of benefiting the fitness or reproductive success of all organisms involved Increased reproductive success may be indirect, via the reproductive success of kin.

This gets into territory that Jane and Becky deliberately avoided with the multi-org branch:  making whole sub-branches based on the benefit to one or other of a pair of animals (parasitism vs symbiosis).  This also strikes me as being _very_ high-level.  What's actually happening in these mutants that mutant cells have an increased chance of forming spores.   Presumably annotation should reflect this.

**Comment by pfey03** [comment]
Wow, love it when you logical outside people discuss Dicty, not having all the history on your back. Not sure in what order I should comment, here is a try.

I think a really big part of the problem is the different purpose of development. At least in all metazoan organisms, and then some, development is part of reproduction. Not so in Dicty as I often described, in return it is a response to starvation. This is the root problem, when thinking as a Dicty biologist, which blurs the lines.

In our anatomy ontology, we do have the relationship ‘develops_from and statements e.g. reading like: ‘mound’ develops_from some ‘loose aggregate’ and ‘migratory slug’ develops_from some ’standing slug’

> Question: Should the term 'sorocarp development' encompass the cell-cell signaling event(s) that lead to aggregation. Do you have terms for this already?

In the GO, if I understand it right, it’s not as ‘easy’ as in the anatomy ontology. Could the cell-cell signalling and thus response to starvation be part_of sorocarp development instead of is_a between all?

The question if Dicty is considered one organism from the aggregate to sorocarp (fruiting body)  stage, is a tricky one. I turned again to our anatomy ontology to determine.

Regarding cheating: From a developmental point , selection who becomes spore and stalk happens very early, during aggregation. In the anatomy ontology we have terms like ‘aggregation territory’ and then ‘loose aggregate’ (def: First adherent mass of cells observed during development, relatively flat with indistinct borders.). During these early stages cells start expressing genes and sorting out.  The term ‘mound’ is already defined as: ‘Hemispherical structure composed of post-aggregative cells that are undergoing differentiation.’  It is of course not as clear cut as in these refs, sorting starts already in the early mound, people showed that with staining of surface antibodies. From the mound (aggregate) stage, the next is the ‘tipped mound’, by which time at least big part of the prestalk have been decided.. The terms through the slug stage have a broad synonym : ‘pseudoplasmodium’.

Thus, if a pseudoplasmodium is one single organism, then it starts after the loose aggregate stage, when the mound is formed. Cheaters rare decided by the time the mound has formed, and they never end up in the tip of the ‘tipped mound’ (tip is pre stalk). You can browse the annotomy ontology here: http://www.ebi.ac.uk/ols/ontologies/ddanat

> social co-operation: "Communication between organisms resulting participation of all communicating organisms in a multi-organism process that ... "

This sound very complicated to me and the focus on multi organism seems misleading a bit

> "... has an equal chance of benefiting the fitness or reproductive success of all organisms involved Increased reproductive success may be indirect, via the reproductive success of kin.

Seems wrong again. Leave reproduction out! This should be a term that really fits for Dicty.

I hope this clarifies things a bit more from a biology standpoint.

Thanks!!

**Comment by dosumis** [comment]
Hi Petra,



Can you suggest a definition of social co-operation (perhaps 'cell co-operation') that doesn't involve multiple organisms or reproduction?  

Given that we seem to be getting a bit lost in abstraction, can you suggest less general that could be used to annotate genes that when mutated lead to 'cheating' (=increased chance of forming spore cell)

Possible compromise - colony development?

Multicellular organisms (to a good approximation) consist of clonally related cells.  A dicty sorocarp may, OTOH, consist of multiple genetically distinct cells.  So I think it makes more sense to talk of a sorocarp as a colony than an organism.  (This means the cells are considered separate organisms throughout development).

colony of unicellular organisms:  A multicellular structure built by the actions of a population of unicellular organisms.  # This term belongs in PCO
Examples include bacterial colonies, yeast colonies, 

development of a colony of unicellular organisms:  "The process by which a population of unicellular organisms, acting together, develops into a colony" 
EquivalentTo: biological_process that results_in_development_of some colony of unicellular organisms

Perhaps we could find a way to define co-operation within this framework, but I'm not sure how.  
        
The cheating phenotype that you intend to infer annotations from seems to be to do with the success in forming spores at the expense of others in the colony.  This looks like (asexual) reproductive success to me.  I don't have any ideas for how else to describe it if it is not.

**Comment by dsiegele** [comment]
I've been thinking about this from the point of view of Myxococcus xanthus-one member of a group of bacterial species where individual cells aggregate and form a fruiting body in response to starvation. In the laboratory Myxo can be grown as individual cells. In its natural environments, Myxo typically exists in populations that coordinate movement and feeding (PMID:12448714).

The definitions of reproduction in GO, and elsewhere, all include "the formation of new individuals". From this point of view, the production of spores in Myxo, and I think also in Dicty, wouldn't be reproduction because the spores aren't new individuals. Instead, they are the subset of cells in the population that differentiated into spores. The spores formed by Myxo and Dicty are survival/dispersal forms. (Endospores formed by Gram-positive bacteria would also be dispersal/survival forms, although their formation doesn't involve fruiting bodies.)

**Comment by pfey03** [comment]
@dsiegele, 
Thanks for all the examples and yes, I've been saying multiple times, the Dicty fruiting / sporulation is differentiation and not reproduction. Please look at my suggested Def below, I'd like to restrict it to developing organisms, Myxococcus I hope will fit?

@dosumis
First we should agree on the term: Im do not think 'cell cooperation' reflects anything I want to express with the annotation. Dicty are  'social amoeba' because this social cooperation is a huge part of their survival strategy. The only compromise, if necessary, would be 'cellular social cooperation', or 'social cell cooperation'.

* Term: social cooperation
* Exact synonym: social co-operation
* Narrow synonym: altruism OR altruistic behaviour  (or would THAT be a good term??)

Suggested Def:
" The process of competitive cell differentiation and development with the outcome of specialised sterile cells or structures that support and protect viable dormant cells."

This expresses well that there might be cheating, IMO.

Thanks!

**Comment by dosumis** [comment]
I've given this some more thought, and think (hope!) I can now cut through the confusions an propose a solution.

Starting points:

1. Behavior (in the general sense) is something that organisms do, not cells in multicullar organisms (people occassionally talk of 'cell behavior' in 
2. Reproduction involves formation of a new individual organism.  In asexual reproduction, that individual is genetically identical to its (one) parent.
3. In multicellular organisms all cells are typically clonal*.  A sorocarp may be composed of genetically diverse cells as long as those cells share some basic compatibility.

(* Ignoring grafts in plants and presence of cells from mother / twins in some animals.)

If we treat a developing sorocarp as a colony of individual organisms then your community's use of the term behavior and reluctance to use the term 'reproduction' make prefect sense.  Individual cells within a developing sorocarp behave in particular ways.  When a cell forms a spore and that spore is dispersed and starts a new cycle, no new individual is formed.

If we treated a sorocarp a developing sorocarp as a single multicellular organism,  each dispersed spore cannot be the same individual as the sorocarp - so spore formation and dispersal *would* count as asexual reproduction.

### Proposal:

colonial development:  development of a colony of unicellular organisms.  A developmental process where the participating cells are unicellular organisms. Examples include the sorocarp development in Dictystelia and  Fruiting body formation in Myxococcus xanthus colonies.
is_a: developmental process
is_a: intraspecies interaction between organisms

colonial development leading to dispersal:  Any process of colonial development leading to production of a fruiting body: a structure that contains spore cells and facilitates their dispersal.
comment: As each cell in a colony is an individual organism, no new organisms are formed on dispersal of cells.  This is therefore not asexual reproduction.

sorocarp development 
is_a: colonial development leading to dispersal

cheating during colonial development leading to dispersal: Any process by which a unicellular organism increases its chances of becoming a spore cell during colonial development leading to dispersal.
is_a: intraspecies interaction between organisms
relationship: part_of colonial development leading to dispersal

This last term feels a bit iffy in that we'd usually treat genetic differences that lead to improved survival / dispersal as phenotype.   But I'll add anyway.

Does this work OK?

**Comment by dosumis** [comment]
Rather than being a superclass of the whole process of "sorocarp development" (which is a clear TPV) I'll switch to:

'aggregation involved in sorocarp development'  is_a: 'cellular response to starvation'

There's plenty of scope for recording cell-cell signaling process terms into the partonomy of sorocarp development, either within the ontology or in annotation extensions / LEGO.

e.g. 

aggregation involved in sorocarp development --- rename --> aggregation leading to sorocarp development

New term: 'cell-cell signaling involved in aggregation leading to sorocarp development'

**Comment by dosumis** [comment]
I suspect this might still need tweaking to avoid terminological confusion:

colonial development leading to dispersal: Any process of colonial development leading to production of a fruiting body: a structure that contains spore cells and facilitates their dispersal.

**Comment by pfey03** [comment]
Does this throw 'social cooperation' out the window? I do not only want to annotate cheaters (cheater genes) with this term, but also those identified compatibility receptors that must fit in order to aggregate, etc. 

> If we treat a developing sorocarp as a colony of individual organisms then your community's use of the term behavior and reluctance to use the term 'reproduction' make prefect sense. Individual cells within a developing sorocarp behave in particular ways. When a cell forms a spore and that spore is dispersed and starts a new cycle, no new individual is formed.

Yes for Dicty

> If we treated a sorocarp a developing sorocarp as a single multicellular organism, each dispersed spore cannot be the same individual as the sorocarp - so spore formation and dispersal would count as asexual reproduction.

No 

This is now a clear cut decision thanks to the many month long debate. 

In sum, your colonial development terms to add some shared parents between Dicty and e.g. Myxocossus, are useful for separation from reproductive processes. 

However I still would like to add 'social cooperation'. To be applicable to all could be part_of  'colonial development leading to dispersal' ?

**Comment by pfey03** [comment]
I finally looked at all my social behaviour annotations again and re-read an important paper that lead me to most of those annotations. (PMID:18272966)

First of all, mutations happen during Dicty growth (what we call 'vegetative stage' and is 'asexual reproduction'). Thus when cells starve and fruit, if cheater mutant cells co-aggregate, they preferably become spores. Start of 2nd paragraph of above paper says: "Clones with mutations in cooperation genes would cheat on the wild type during the social stage."

In the screen, they found many what they call 'facultative cheaters' who develop normally when clonal, in comparison to 'obligate chatters, who have developmental defects when clonal). especially in this focus on facultative cheater mutants, they use the term 'cooperation genes'.

None, neither the obligate nor the facultative cheater genes should be annotated with 'cheating during colonial development leading to dispersal' (this term not needed). The cheating is a phenotype but what is really at stake is different levels of 'social cooperation'.

Thanks,
Petra

**Comment by dosumis** [comment]
Changes so far:



New term:

Changes made so far:



Some of these terms were under 'reproductive development'.  I removed this classification. 

New term: 

![image](https://cloud.githubusercontent.com/assets/112839/21028567/9e844ba4-bd8d-11e6-8448-8e6105270743.png)

![image](https://cloud.githubusercontent.com/assets/112839/21028578/aa25dc48-bd8d-11e6-8ee2-3e69e5688e93.png)

'sorocarp dev' now only has one parent:

![image](https://cloud.githubusercontent.com/assets/112839/21028590/c1a41484-bd8d-11e6-9c4e-9949705c2404.png)

aggregation is now a cellular response to starvation:

![image](https://cloud.githubusercontent.com/assets/112839/21028609/dde66b92-bd8d-11e6-908c-2d86b4673ae6.png)

Also removed a bunch of classifications under reproductive process to parts of sorocarp development.

**Comment by pfey03** [comment]
Once all this sorocarp development is away from reproduction and this structure is clean, when will I finally get my new term 'social cooperation'? Let me know if I need to look at new development structure/graphs when officially available, and then think again where the term could go. Off next week until January 3rd. Happy holidays!

**Comment by dosumis** [comment]
> None, neither the obligate nor the facultative cheater genes should be annotated with 'cheating during colonial development leading to dispersal' (this term not needed). The cheating is a MUTANT phenotype but what is really at stake is different levels of 'social cooperation'.

I don't think it's so clear cut.  From the paper: "Cheating occurs in chimaeras between wild isolates, and chimaeras are found in the wild, suggesting that cheating occurs in nature."

### My reading of the paper:  

Socially co-operative development by aggregation often leads to formation of a chimeras:  structures consisting of cells of multiple genotypes.  In the case of chimeric sorocarp development, individuals of one genotype can gain a selective advantage over individuals of another if they 'cheat': increase their chances of forming spore cells at the expense of cells of a different genotype.   The existence of cheating puts evolutionary pressure on the maintenance of socially co-operative development as a strategy.  

Cheating is a phenotype and likely to be a relative one: cells of genotype A may have a competitive advantage in forming spore cells in chimeras with cells of genotype B but not with cells of genotype C. In this paper they find mutations that lead to 'cheating' when cells carrying the mutation form a chimeric sorocarp with otherwise genetically identical cells - i.e. they test against one background.  The mechanisms by which cheating occurs vary, as shown by experiments looking at the location of cells from different cheating mutants.  All this makes annotation based on these experiments a bit of a stretch for GO, but I think I can see a way to do it (see below).

Upthread you wrote: 

>  I do not only want to annotate cheaters (cheater genes) with this term, but also those identified compatibility receptors that must fit in order to aggregate, etc.

We need to distinguish between meanings of 'co-operative' in this discussion.  

1. Some level of co-operation is required for any colonial development, including colonial development in which cheating occurs.  For example, compatible compatibility receptors are required for aggregation in the first place (You should probably have a term for these).
2. 'Co-operative' is sometimes used to refer to sorocarp development in which all cells have an equal chance of forming a spore cell.
3. Co-operative development as an evolutionary strategy is threatened by the presence of cheaters - so cheater 'mutations' identify genes that, in their 'wild-type' state are required for the long term evolutionary viability of co-operative development.   This use is clearly out of scope for GO. 

I think it's worth making these distinctions clear.  If no objections, I'll go with this solution.

'colonial development' exact synonym: 'socially co-operative development'
.   <- is_a: sorocarp development
.   .   <- is_a 'chimeric sorocarp development': "Development of a sorocarp formed by aggregation of cells with different genotypes." 
.   .   .   <- is_a 'chimeric sorocarp development - altruistic': "Development of a chimeric sorocarp in which cells of all genotypes have an equal chance of becoming a spore cell." exact synonym: fully co-operative chimeric sorocarp development
.   .   .   .   <- part_of 'cheating during chimeric sorocarp development':  Any process during chimeric sorocarp development by which a cell increases the number of spore forming cells sharing its genotype at the expense of cells of other genotypes.'
.    .    .    .    <- is_a { could potentially enumerate mechanisms under here  }

is_a: regulation of sorocarp spore cell differentiation
.   .   <- is_a 'cheating during chimeric sorocarp development'


TBA: Please make a separate ticket for compatibility receptors / cell adhesion during sorocarp aggregation.   

(This seems to have become a bit of an essay - but I think we're nearly done!)

**Comment by dosumis** [comment]
Class hierarchy only:

![image](https://cloud.githubusercontent.com/assets/112839/21684931/8ae57fb6-d357-11e6-8d1d-3ca8a3182ca4.png)

Note 'colonial development' has exact synonym: 'socially co-operative development'

**Comment by pfey03** [comment]
Thanks for all your work David, and sorry, but I was off for holidays and got sick, so was out a bit. 

 I argued above that cheating is a mutant phenotype, but maybe because I have all that history on certain mutants in my head. With that cited new paper it does seem a more wide-spread phenomenon and thus kin recognition and cheating are in balance of some sort. Thus, thanks for the cheating terms, will see how I can use them.

'colonial development' as a primary term does nothing for Dicty people, and the child 'aggregation...' is also used in contexts other than social behaviour studies, so I still do not have any term for what I originally wanted. Will see how the 'altruistic chimeric development' will work out.

In sum, glad you are happy as ontologist, it's not what I originally had in mind but will see how to use these new terms, and if that satisfies me as a representative of Dicty researchers

Thanks again!
Petra

**Comment by pfey03** [comment]
The term GO:0099120 colonial development, I saw the term was added mid December. Then, a few days later, you added the synonym "socially co-operative development". The Def is fine.

So why is not "socially co-operative development" the PRIMARY term? (Also cooperation is not hyphenated in US and thus needs to be EXACT synonym)

The cited Myxococcus paper has the title "Experimental **social evolution** with Myxococcus" xanthus.  
Thus 'social cooperation' or 'socially cooperative development'  should be an acceptable primary term for the Myxococcus folks?

I used the cheating term for many social evolution annotated genes I had, however, for this paper:
PMID:23910661 "Kin Recognition Protects Cooperators against Cheaters". Kin recognition and cheating has been annotated before to the involved genes/proteins, but for this paper I liked using just social behavior, or now 'socially cooperative development'. 

Please make 'socially cooperative development' / socially co-operative development the primary term of  GO:0099120. 

No Dicty person would ever understand what's meant by looking at 'colonial development'. I cannot use it and I'm stranded like before for some annotations - when I want to describe which proteins make Dicty a social amoeba, not just specifically kin recognition or cheating.

Thanks!

**Comment by dosumis** [comment]
Hi Petra,

> Please make 'socially cooperative development' / socially co-operative development the primary term of GO:0099120.

Done.

> I used the cheating term for many social evolution annotated genes

You might want to consider the altruistic chimeric sorocarp dev term for some of these too (the relative phenotype thing makes this an inherently hard problem, but you may be able to come up with some rules of thumb.

>  I had, however, for this paper:
PMID:23910661 "Kin Recognition Protects Cooperators against Cheaters". Kin recognition and cheating has been annotated before to the involved genes/proteins, but for this paper I liked using just social behavior, or now 'socially cooperative development'.

'socially cooperative development' is the root term of your whole development hierarchy, so [anything involved in Dicty dev should be directly or indirectly annotated to it](http://www.ebi.ac.uk/QuickGO/GAnnotation?a=&goid=GO%3A0099120&termUse=ancestor&relType=IPO%3D&customRelType=IPOR%2B-%3F%3D&evidence=IMP%2CIGI%2CIPI%2CIDA%2CIEP%2CEXP&protein=&tax=33083&qualifier=&ref=&with=&source=&q=&col=proteinDB%2CproteinID%2CproteinSymbol%2Cqualifier%2CgoID%2CgoName%2Caspect%2Cevidence%2Cref%2Cwith%2CproteinTaxon%2Cdate%2Cfrom%2Csplice&select=normal).

I'd be happy to add a term for kin recognition.  We should be able to come up with something nice and specific that puts this in the context of co-operative dev.

**Comment by pfey03** [comment]
> Done.

Many thanks!

> You might want to consider the altruistic chimeric sorocarp dev term for some of these too (the relative phenotype thing makes this an inherently hard problem, but you may be able to come up with some rules of thumb.

I will.Maybe add for these, but since it's a genomic screen for cheaters I usually do not over annotate. But will reconsider of course. Need to adjust our phenotype ontology too, so still cleaning up.

> I'd be happy to add a term for kin recognition.  We should be able to come up with something nice and specific that puts this in the context of co-operative dev.

I got this term a while back, the synonym is kin recognition. It's so far not in relation to cooperative development. Leave it up to you to decide if it's just easy to add some different parentage.
http://amigo.geneontology.org/amigo/term/GO:0097656

Thanks again!

**Comment by pfey03** [comment]
regarding kin recognition: Here is the the original request:
https://github.com/geneontology/go-ontology/issues/11360

**Comment by pfey03** [comment]
Saw in AmiGO that the term/synonym swap for GO:0099120.has been implemented. Should make it's why to QuickGO and P2GO soon. Thanks for everything! 
If you want to restructure GO:0097656 - cell-cell self recognition / kin recognition, maybe you open another ticket? Or let me know here.
From my side and all other aspects, this ticket can otherwise be closed.

**Comment by pgaudet** [comment]
@dosumis @pfey03 
Should we close this ?

**Comment by dosumis** [comment]
>  @dosumis @pfey03
> Should we close this ?

I think so.  Major issues dealt with.  Socially co-operative development now covers what Petra was using 'behavior' for (I think) + I ended up doing a pretty big overhaul of terms applicable to dicty dev as the ontology was pretty broken.  Still worth rephrasing the def of behavior to make restriction clear.   Maybe open a new ticket for that and refer back?
